pkgs/desktops/gnome-3/extensions/switcher/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, glib, gnome3 }:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-shell-switcher";
+  version = "master";
```

Comment:
Version number should be unstable-XXX-XX-XX. The X's are the date of the commit.